src/core/macro.scala

```diff
@@ -100,12 +100,13 @@ object Macro:
     import qctx.reflect._
 
     def normalizedName(s: Symbol): String = if s.flags.is(Flags.Module) then s.name.stripSuffix("$") else s.name
+    def normalizedFullName(s: Symbol): String = if s.flags.is(Flags.Module) then s.fullName.stripSuffix("$") else s.fullName
```

Comment:
Since this is a full path, there might be $ in other positions than the last.

Example from caliban test suite:
expected:
`caliban.TestUtils.InvalidSchemas.Interface.ClashingObject`
actual:
`caliban.TestUtils$.InvalidSchemas$.Interface.ClashingObject`

Reply:
Makes sense, see if it works for You now :slightly_smiling_face:

Reply:
Yep, looks good now 👍